The results of running the command line `mvn clean install` in `jetty-ee10\jetty-ee10-osgi\test-jetty-ee10-osgi`.

```
[ERROR] Failures:
[ERROR]   TestJettyOSGiBootWithAnnotations.testIndex:100 Response status code expected:<200> but was:<503>
[ERROR]   TestJettyOSGiBootWithBundleJettyHome.testJspDump:83 expected:<200> but was:<503>
[ERROR]   TestJettyOSGiBootWithJsp.testJspDump:83 expected:<200> but was:<503>
[ERROR]   TestJettyOSGiBootWithWebSocket.testWebsocket:99
[ERROR]   TestJettyOSGiClasspathResources.testWebInfResourceNotOnBundleClasspath:93 expected:<200> but was:<503>
[ERROR] Errors:
[ERROR]   TestJettyOSGiClasspathResources.testWebInfResourceOnBundleClasspath:115 ┬╗ IO Access is denied
[INFO]
[ERROR] Tests run: 7, Failures: 5, Errors: 1, Skipped: 0
```

I cannot get specific DEBUG output, as the `jetty-ee10\jetty-ee10-osgi\test-jetty-ee10-osgi\README.txt` doesn't work.
The jetty-slf4j-impl isn't present in the paxexam environment, so the jetty-logging.properties recommendations is not producing a change in output (for oexample, I waned to see the DEBUG of Jetty startup)

```
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
```

--- a/jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebInfConfiguration.java
+++ b/jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebInfConfiguration.java
@@ -239,7 +239,8 @@ public class WebInfConfiguration extends AbstractConfiguration
 
                 if (war != null)
                 {
-                    Path warPath = Path.of(war);
+                    Path warPath = Path.of(URIUtil.toURI(war));
+                    
                     // look for a sibling like "foo/" to a "foo.war"
                     if (FileID.isWebArchive(warPath) && Files.exists(warPath))
                     {

